giddy-banana/models/comment.cjs

```diff
@@ -0,0 +1,4 @@
+module.exports = (db) => {
+  db.Article.belongsToMany(db.User, { through: "Comment" });
+  db.User.belongsToMany(db.Article, { through: "Comment" });
+};
```

Comment:
What do you think about having a full fledged model for this available via db?

Reply:
This double export is not going to work. You set the default export of this module twice.

Reply:
I think you probably wanted to remove the above lines.